jdaviz/configs/default/plugins/subset_tools/tests/test_subset_tools.py

```diff
@@ -234,6 +233,21 @@ def test_import_spectral_regions_file(cubeviz_helper, spectrum1d_cube, tmp_path)
         plg.combination_mode = 'test'
 
 
+def test_import_sky_region_in_cubeviz(cubeviz_helper, spectrum1d_cube):
+    cubeviz_helper.load_data(spectrum1d_cube)
+    plg = cubeviz_helper.plugins['Subset Tools']
+
+    ra = 339.0149557 * u.deg
+    dec = 33.97591 * u.deg
+    coord = SkyCoord(ra=ra, dec=dec, unit=u.deg)
+    aper_rad = 0.5 * u.arcsec
+
+    aperture = CircleSkyRegion(coord, aper_rad)
+
+    status = plg.import_region(aperture, return_bad_regions=True)
+    assert status == []
```

Comment:
Following Cami's comment. I don't think this one assert check is enough. Please add a check to see if Subset 1 is really created and whether the parameters are correct. Thanks!